### DISCUSSION TITLE
Day statistics are lagging behind

### DISCUSSION THREAD

**Discussion by digiater** [dsmrreader/dsmr-reader#1476] [Questions & answers / How-to's] [answered]
DSMR-reader v4.16.2

De volgende melding staat op de 'about' pagina:

      Day statistics are lagging behind   (1 day, 9 hours ago)

Ik moet vermelden dat gisteren de gasmeter is verwijderd omdat we van het gas af zijn. En dat de update van die dag dus ergens fout loopt, waarschijnlijk omdat er geen gas gegevens meer worden doorgegeven.

In de logs directory van dsmr-reader staat alleen oude data van vorig jaar.

Wat kan er aande hand zijn?

Jur.

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je melding. De kans is groot dat het inderdaad ligt aan dat de gasgegevens "opeens" ontbreken voor DSMR-reader. 

Deze detecteert helaas niet automatisch dat de meter uberhaupt weg is, maar ik denk dat je het wel makkelijk kan fixen via een vinkje in de admin. 
Bij de backend-instellingen op de pagina `/admin/dsmr_backend/backendsettings/` kun je een vinkje aanzetten waarmee gas-verbruik genegeerd wordt. Ik hoor graag of dat het voor je oplost.

**Comment by digiater** [comment]
Dat lost het inderdaad op. Bedankt!

Jur.

**Comment by cranberryNL** [comment]
Bij ons is vorige week de gasmeter verwijderd en ik loop tegen dezelfde melding aan. In de backend configuration kan ik de optie 'Disable gas capability' activeren. Gevolg daarvan is echter dat ik m'n historisch gasverbruik van de achterliggende jaren niet meer kan zien.
Als ik trouwens genoemde optie weer uitschakel, zie ik wel het historisch gasverbruik wel weer zichtbaar. Blijft dat zo of wordt dat een keer opgeschoond?
Is er een oplossing voorhanden?

**Reply by dennissiemensma** [reply]
Ik heb helaas momenteel geen oplossing voor je die beide doet. Het uitschakelen en toch terug kunnen kijken is een combinatie waar ik geen rekening mee heb gehouden.
